src/sync/deployment.test.ts

```diff
@@ -426,14 +483,14 @@ describe("sync/deployments", () => {
 				"deploymentSequenceNumber": 500226426,
 				"updateSequenceNumber": 500226426,
 				"displayName": "[TEST-123] test-commit-message",
-				"url": "https://github.com/test-repo-owner/test-repo-name/commit/51e16759cdac67b0d2a94e0674c9603b75a840f6/checks",
+				"url": null,
 				"description": "deploy",
 				"lastUpdated": "2022-02-03T22:45:04.000Z",
-				"state": "successful",
+				"state": "unknown",
```

Comment:
This is the existing bug once I update the graphql json fixture to be more "close" to real world.